src/js/dropdown-menu/dropdown-menu-view.js

```diff
@@ -42,6 +42,8 @@ define([
         var DropdownMenuView = Backbone.View.extend({
             tpl: _.template(DropdownTpl),
 
+            // For event handling only instances do I need to scope this somehow?
```

Comment:
These events will only be scoped to the root element of the view:

http://backbonejs.org/#View-events

I think that's what you want. If so, you can just remove this comment.